lib/db/schema/patch-034-035.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  unblockCode VARCHAR(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  UNIQUE INDEX unblockCodes_uid_code(uid, unblockCode)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createUnblockCode_0` (
+    IN inUid BINARY(16),
+    IN inCode VARCHAR(32),
+    IN inCreatedAt BIGINT SIGNED
+)
+BEGIN
+    INSERT INTO unblockCodes(
+        uid,
+        unblockCode,
+        createdAt
+    )
+    VALUES(
+        inUid,
+        inCode,
+        inCreatedAt
+    );
+END;
+
+CREATE PROCEDURE `consumeUnblockCode_0` (
+    inUid BINARY(16),
+    inCode VARCHAR(32)
+)
+BEGIN
+    CREATE TEMPORARY TABLE IF NOT EXISTS unblockTimestamps AS
+    (
+        SELECT createdAt FROM unblockCodes
+        WHERE
+            uid = inUid
+        AND
+            unblockCode = inCode
+    );
```

Comment:
I remember trying several different things to get the `affectedRows` to work in `db.js`. I ended up just checking the results length instead, so this part shouldn't affect it. Nothing I can find suggests one is better than the other.